management/virtualmachine/resourceextensions_test.go

```diff
@@ -0,0 +1,38 @@
+package virtualmachine
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management/testutils"
+)
+
+func init() {
+	flag.BoolVar(&extraVerbose, "resourceextensions.verbose", false, "List all resource extensions")

```

Comment:
No, I haven't been able to get `go test` to list all flags for the package under test. You have to build the test binary with `go test -c .` and then execute the test binary with --help (`virtualmachine.test.exe --help`). I think this makes the ability for tests to declare extra flags very hard to discover and in most cases not the best practice (as you pointed out).